api/channel.go

```diff
@@ -901,14 +911,21 @@ func updateLastViewedAt(c *Context, w http.ResponseWriter, r *http.Request) {
 		go clearPushNotification(c.Session.UserId, id)
 	}
 
-	preference := model.Preference{
+	chanPref := model.Preference{
 		UserId:   c.Session.UserId,
-		Category: model.PREFERENCE_CATEGORY_LAST,
+		Category: c.TeamId,
```

Comment:
nop, I need the last channel for every team that you are a member of and not just the last channel you saw